docs/UserGuide.adoc

```diff
@@ -67,6 +68,11 @@ Examples:
 * `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
 * `add Betsy Crowe pp/1234567 e/betsycrowe@gmail.com pa/Newgate Prison t/criminal t/friend`
 
+== Sort AddressBook lists : `sort`
```

Comment:
That's good that you have updated the User Guide (UG) to show the newly implemented function that you have created. The UG is the place where the user will go to find out the different features and the usage of it most of the time.